docs/Schema.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE card(
+    id INT PRIMARY KEY,
+    study_id INT,
+    front TEXT,
+    back TEXT
+);
+
+CREATE TABLE study_set(
+    id INT PRIMARY KEY,
+    owner_userid INT,
+    title TEXT,
+    description TEXT,
+    university_id INT,
+    professor TEXT,
+    academic_time_period TEXT,
+    course_name TEXT,
+    creation_time DATE,
+    update_time DATE
+);
+
+CREATE TABLE university(
+    id INT PRIMARY KEY,
+    name TEXT,
+    state TEXT,
+    url  TEXT
```

Comment:
How will this url be used?

Reply:
This is the website of the university. We are thinking of adding a feature in the future to have a link to take the user to this website. We are keeping it here since the website we want to scrap from has each university website as part of its table field so it should not be hard to get the url.

Reply:
I doubt that providing a link to university's website homepage is a worthy feature for this project though...It's one google search away with the university name and it won't contain any useful information to the course.

Could this be removed for now? You can always add it back if needed. It's probably a good idea to leave out fields that won't be used for a while when starting out.

Reply:
Okay, sure.